inspirehep/config.py

```diff
@@ -1188,3 +1188,9 @@ def _(x):
     "stat.OT": "Other",
     "stat.TH": "Other"
 }
+
+# Import local config file if it is present.
+try:
+    from inspirehep.config_local import *
```

Comment:
It's what I do in HEPData too. I think it's more obvious than having inspirehep.cfg in the var/instance dir since people normally forget about it.